src/libpriv/rpmostree-core.cxx

```diff
@@ -2173,9 +2127,7 @@ rpmostree_context_prepare (RpmOstreeContext *self,
       }
   }
 
-  g_assert (!self->rojig_spec);
-
-  if (!self->rojig_pure)
+  if (TRUE)
```

Comment:
Personally OK with incurring the messier diff/git history of dedenting this block FWIW.